app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,9 @@
   </head>
 
   <body>
+    <%= render(NavComponent.new) %>
+    <%= render(TickerListComponent.new(stocks: [{name: "Netflix", ticker: "NFLX", price: 115, change: 0.033}, {name: "Sony Corporation", ticker: "SNE", price: 58.99, change: -0.01}, {name: "Under Armour", ticker: "AU", price: 18.07, change: 0.79}, {name: "Ford", ticker: "F", price: 9.75, change: -0.03}])) %>
```

Comment:
This temporary hard-coded stock collection may better live directly in the view object, instead of in the view template.

Reply:
Done, and added some testing